encoder/vaapiencoder_h264.cpp

```diff
@@ -1319,9 +1321,10 @@ bool VaapiEncoderH264::addSliceHeaders (const PicturePtr& picture) const
         sliceParam->slice_qp_delta = initQP() - minQP();
         if (sliceParam->slice_qp_delta > 4)
             sliceParam->slice_qp_delta = 4;
-        sliceParam->slice_alpha_c0_offset_div2 = 2;
-        sliceParam->slice_beta_offset_div2 = 2;
 
+        sliceParam->disable_deblocking_filter_idc = !m_videoParamAVC.enableDeblockFilter;
+        sliceParam->slice_alpha_c0_offset_div2 = m_videoParamAVC.deblockAlphaOffset / 2;
```

Comment:
@wudping and @lizhong1008 
Just checked spec, slice_alpha_c0_offset_div2 is from spec, do we need change m_videoParamAVC.deblockAlphaOffset to sliceAlphaC0OffsetDiv2? Because deblockAlphaOffset is not so meaningful,  2n and 2n+1 will get same result for deblockAlphaOffset